javalib/src/main/scala/java/nio/channels/spi/SelectorProvider.scala

```diff
@@ -0,0 +1,25 @@
+package java.nio.channels.spi
+
+import java.nio.channels._
+import java.net.ProtocolFamily
+
+abstract class SelectorProvider protected () {
+
+  def openDatagramChannel(): DatagramChannel
+  def openDatagramChannel(family: ProtocolFamily): DatagramChannel
+  def openPipe(): Pipe
+  def openSelector(): AbstractSelector
+  def openServerSocketChannel(): ServerSocketChannel
+  def openSocketChannel(): SocketChannel
+
+  // There is no JVM, so there is no way to inherit this, hence null
```

Comment:
At the risk of bikeshedding, I suggest dropping the
, so there is no way to inherit this".

Not critical, just more positive & enabled.